inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -1382,6 +1391,14 @@ export class AnimationCurveEditorComponent extends React.Component<
     }
   }
 
+  isCurrentFrame(frame: number) {
+    if (this.state.currentFrame === frame) {

```

Comment:
(nit) `return this.state.currentFrame === frame` will be cleaner

Reply:
gotcha!